addons/project/report/project_report_views.xml

```diff
@@ -17,7 +17,7 @@
             <field name="arch" type="xml">
                 <graph string="Tasks Analysis" type="bar" sample="1" disable_linking="1">
                      <field name="project_id" type="row"/>
-                     <field name="user_id" type="col"/>
+                     <field name="stage_id" type="col"/>
```

Comment:
I am not sure that it is feasible or not but we should try to keep the field in the arch and making it `unusable` (invisible="1"?). By doing so we could ensure that no custo break.

Reply:
Thanks for the great tip !

It will work thanks to : 
https://github.com/odoo/odoo/blob/saas-14.4/addons%2Fweb%2Fstatic%2Fsrc%2Fjs%2Fviews%2Fgraph%2Fgraph_view.js#L104-L116